src/admin/site_environments.rst

```diff
@@ -0,0 +1,146 @@
+Site Environments
+=================
+
+The site environments required for running PanDA jobs.
```

Comment:
Wording suggestion:  At many sites multiple experiments share the computing resource.  Thus experiment specific settings should not be set globally.  Instead, these settings can be put in a file that will be sourced by the PanDA pilot as well as the user.  For an complete example of the file, see section X-link-here.  The next section will describe the content of the file.